tutorials/cloud-iot-rtdp/streaming/pom.xml

```diff
@@ -0,0 +1,146 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
FYI this file is tab-delimited and the others are space-delimited. You may want to search / replace the tabs with spaces.